scripts/custom/clean_dateTime.py

```diff
@@ -0,0 +1,74 @@
+import datetime
+
+from scripts import utils
+
+
+def clean_dateTime(raw_input):  # noqa: C901
+    if not isinstance(raw_input, str):
+        raw_input = str(raw_input)
+    if utils.is_empty(raw_input):
+        return ""
+
+    date = None
+
+    # Handle YYYY
+    try:
+        date = datetime.datetime.strptime(raw_input, "%Y")
+        result = date.strftime("%Y")
+    except ValueError:
+        pass
+
+    # Handle YYYY-MM
+    try:
+        date = datetime.datetime.strptime(raw_input, "%Y-%m")
+        result = date.strftime("%Y-%m")
+    except ValueError:
+        pass
+
+    # Handle YYYY-MM-DD
+    try:
+        date = datetime.datetime.strptime(raw_input, "%Y-%m-%d")
+        result = date.strftime("%Y-%m-%d")
+    except ValueError:
+        pass
+
```

Comment:
Can you add the case for YYYYMMDD and YYYYMM also?

Reply:
Done